hyp3_gamma/__main__.py

```diff
@@ -213,6 +221,13 @@ def water_map():
             f'--hand-threshold {args.hand_threshold} --hand-fraction {args.hand_fraction} '
             f'--membership-threshold {args.membership_threshold}', uselogging=True)
 
+    if args.include_flood_depth:
+        execute(f'conda run -n  asf-tools flood_map {product_name}/{product_name}_FM.tif '
+                f'{product_name}/{product_name}_WM.tif {product_name}/{product_name}_WM_HAND.tif '
+                f'--estimator {args.estimator} --water_level_sigma {args.water_level_sigma} '
+                f'--known_water_threshold {args.known_water_threshold} --iterative-bounds {args.iterative_bounds} ',
```

Comment:
now that `args.iterative_bounds` is a list, I think we need to do a little manual formatting here. As written, we'd get `--iterative-bounds [0, 15]`, where I think we need `--iterative-bounds 0 15`